classes/Http/Controller/Admin/SpeakersController.php

```diff
@@ -209,20 +213,14 @@ public function demoteAction(Request $request)
 
     public function promoteAction(Request $request)
     {
-        /* @var AccountManagement $accounts */
-        $accounts = $this->service(AccountManagement::class);
-
         $role = $request->get('role');
         $id   = (int) $request->get('id');
 
-        /** @var Session\Session $session */
-        $session = $this->service('session');
```

Comment:
Since we would otherwise be invoking `$request->getSession()` multiple times, can we maybe just use `$session` here and assign it `$request->getSession()`?

Reply:
See the discussion above.

Reply:
See the discussion above.